dev/how-to/maintain/tidb-in-kubernetes/auto-failover.md

```diff
@@ -0,0 +1,86 @@
+---
+title: Automatic Failover Policies of TiDB Cluster Components on Kubernetes
+summary: Learn the automatic failover policies of TiDB cluster components on Kubernetes.
+category: how-to
+---
+
+# Automatic Failover Policies of TiDB Cluster Components on Kubernetes
+
+Automatic failover means that when some nodes in the TiDB cluster fail, TiDB Operator automatically adds a new node to ensure the high availability of the cluster like the `deployment` behavior that happens in Kubernetes.
+
+TiDB Operator manages Pods based on `StatefulSet`, but `StatefulSet` does not automatically create a new node to replace the original node when some Pod goes down. For this reason, TiDB Operator develops the automatic failover feature and expands the behavior of `StatefulSet`.
+
+The automatic failover feature is disabled by default in TiDB Operator. You can enable it by setting `controllerManager.autoFailover` to `true` in the `charts/tidb-operator/values.yaml` file when deploying TiDB Operator:
+
+```yaml
+controllerManager:
+ serviceAccount: tidb-controller-manager
+ logLevel: 2
+ replicas: 1
+ resources:
+   limits:
+     cpu: 250m
+     memory: 150Mi
+   requests:
+     cpu: 80m
+     memory: 50Mi
+ # autoFailover is whether tidb-operator should auto failover when failure occurs
+ autoFailover: true
+ # pd failover period default(5m)
+ pdFailoverPeriod: 5m
+ # tikv failover period default(5m)
+ tikvFailoverPeriod: 5m
+ # tidb failover period default(5m)
+ tidbFailoverPeriod: 5m
+```
+
+By default, `pdFailoverPeriod`, `tikvFailoverPeriod` and `tidbFailoverPeriod` are set to be 5 minutes, which is the waiting timeout after confirming the instance failure. After this time, TiDB Operator begins the automatic failover process.
+
+## Implementation principles
```

Comment:
Maybe we should title it "Automatic failover policies", and for each component, just use the component name as the header.